### ISSUE TITLE
Sprint 11 Goal

### ISSUE THREAD

**Issue by hmitsch** [mozilla-iam/iam-project-backlog#163]
## Prioritized Themes
> Get things to **DONE-DONE**
1. RP Cutovers
1. [LDAP INTEGRATION - MOZ] _Q2 OKR PROD transition_ (Mozillians/CIS/Discourse) 
1. [MOZILLIANS.ORG]: _Infrastructure and application debt repayment_
1. (bonus) Launch SSO Dashboard for community

### Details
1. [RP Cutover] Migrate sprint batch, [Migration Plan](https://docs.google.com/spreadsheets/d/1VLwh2A9qbaRqByj7BYbnziWQLL_DdjemAiwT4_0OebE/edit)
   - [ ] #130 Group 0 (in execution) - Stage and Dev instances that are managed by IT or InfoSec (6 of 8 completed)
   - [ ] #131 Group 1 (in requirements definition) - Managed within IT (InfoSec, WebOps)
   - @tyearns please add/update details
1. [LDAP INTEGRATION - MOZ] Change Integration Service ([CIS](https://github.com/mozilla-iam/cis/issues)): _Q2 OKR PROD transition_
      - [ ] #116 RP Handle new group information (In progress)
      - [x] Remove Hard Coded Partition Key https://github.com/mozilla-iam/cis/issues/54
      - [x] Remove Duplicate Code https://github.com/mozilla-iam/cis/issues/55
      - [x] Move Logging Utils to CIS Core https://github.com/mozilla-iam/cis/issues/53
      - [ ] Filter users interacted with https://github.com/mozilla-iam/cis/issues/59
      - [ ] Merge demo functionality of CIS to mozillians.org
> Moz/CIS integration (issue #107) - In-progress
> CIS clean-up activities (Prod readiness) (5) - Requires 1 week effort (kang 50%/andrew 100%) - Sprint 11 (118, 124, Back-up/recovery of dynamo)
1. [MOZILLIANS.ORG]: _Infrastructure Cleanup_
   - [x] Infra Maintenance https://github.com/mozilla/parsys/issues/202
   - [ ] Fix mozillians.org S3 permissions https://github.com/mozilla/parsys/issues/134 https://github.com/mozilla/parsys/issues/191
   - [ ] Remove memcached from mozillians group https://github.com/mozilla/parsys/issues/190
   - [ ] Add cdn to mozillians media https://github.com/mozilla/parsys/issues/189
   - [x] Move staging apps to prod cluster https://github.com/mozilla/parsys/issues/133 https://github.com/mozilla/parsys/issues/158
1. [MOZILLIANS.ORG]: _Application Cleanup_
   - [ ] Mozillians.org repo/Github org (debt) cleanup
   - [ ] Spamfiltering: get it to DONE-DONE, i.e. enable automatic account removal
   - [ ] Binary search: get it to DONE-DONE
   - [ ] [Bug 1373894](https://bugzilla.mozilla.org/show_bug.cgi?id=1373894): Potential XSS in Mozillians.org
    - [ ] [Bug 1376416 mozillians.allizom.org and mozillians-dev.allizom.org  are unreachable - 403 forbidden](https://bugzilla.mozilla.org/show_bug.cgi?id=1376416)
    - [ ] [Fix bug 1201027] Simplify Profile status UI https://github.com/mozilla/mozillians/pull/1436
    - [ ] Groups grid UI enhancements https://github.com/mozilla/mozillians/pull/1426/
    - [ ] Implement a way to trigger reindexes https://github.com/mozilla/parsys/issues/201
    - [ ] (Mentoring) [Bug 847911](https://bugzilla.mozilla.org/show_bug.cgi?id=847911) Create a view with all vouched mozillians
    - [ ] (Mentoring) [Bug 1333373](https://bugzilla.mozilla.org/show_bug.cgi?id=1333373) 
Redesign footer based on new brand identity
1. QA Engineering
   - [x] Set RPs login tests to run in Jenkins https://github.com/mozilla/iam-project-backlog/issues/168
   - [x] Implement login test for mozilla.slack.com https://github.com/mozilla/iam-project-backlog/issues/167
   - [x] Implement login test for reps.allizom.org https://github.com/mozilla/iam-project-backlog/issues/166
   - [x] Implement login test for moderator.mozilla.org https://github.com/mozilla/iam-project-backlog/issues/165
   - [x] Set the existing session handling tests to run in parallel https://github.com/mozilla/iam-project-backlog/issues/164
   - [ ] (bonus) Automate client testing in auth0-deploy CI, include SAML clients https://github.com/mozilla/iam-project-backlog/issues/162
1. (bonus)  [SSO Dashboard] Launch for Communities
   - [x] Enable Passwordless email login
   - [x] Applications in grid for public users (Communities): AirMo, Discourse, Moderator, Mozillians, Reps
   - [ ] Communicate the publicly available Dashboard widely: Discourse, various Telegram Channels, mailto all Mozillians (via Salesforce Marketing Cloud), Mozilla Reps

## For mental sanity: Other InfoSec Work
1. [ ] [SECOND OPINION] _PoC for end-to-end testing_ and _productionalizing_ implementation #79 
1. [ ] Implement SSH Federated Access #169 
1. [ ] Accommodate Auth0 OIDC Conformant mode requirements #170 #171
1. [x] [Bug 1332761] Discourse authentication emails taking too long to arrive

## For mental sanity: Other ParSys Work
1. [CoSS](https://github.com/mozilla/coss/) Sprint 11 https://github.com/mozilla/coss/issues/209
1. [DISCOURSE] *Stop starting and start finishing (and start the migration to discourse.mozilla.org)*
   - [ ] Add additional email address support https://github.com/mozilla/discourse/issues/10 [_phase 1 closer to being merged upstream, but not there yet_]
   - [ ] Get All Hands OKR demo in prod [_currently writing tests_]
      - [ ] Implement improved session management https://github.com/mozilla/discourse/issues/12
      - [ ] Sync groups from IAM https://github.com/mozilla/discourse/issues/54
   - [ ] Email-out permission checking https://github.com/mozilla/discourse/issues/13 [_currently writing tests_]
   - [ ] Avoid N+1 issue with pre-email permissions check https://github.com/mozilla/discourse/issues/63 [_blocked by Auth0 API not working as documented, and restructuring required in Discourse core_]
   - [ ] [bug] Email-in thread with same title does nothing https://github.com/mozilla/discourse/issues/68
   - [ ] Set up staging instance https://github.com/mozilla/discourse/issues/70
   - [ ] Migrate to discourse.mozilla.org https://github.com/mozilla/discourse/issues/71

## Unplanned Scope
**Note:** please add any unplanned scope that you worked on here.
* Work on open issues from [Mozillians.org Penetration Test](https://docs.google.com/document/d/19xD3UlDgb_vQJYVX0g3pvqfg1a3O6HjIsnvnC2vg-kI/edit)
* Discourse bug: Missing the end of emails https://github.com/mozilla/discourse/issues/61
* [Bug 1382208](https://bugzilla.mozilla.org/show_bug.cgi?id=1382208) - "Nominate Rep of the Month" task not expired
* [Bug 1381890](https://bugzilla.mozilla.org/show_bug.cgi?id=1381890) - [Basket] Add double opt-in parameter to email subscription API calls & clean up existing Mozillians/NDA subscribers

## Out of Scope / Later Items
Themes which are not **yet** slated for delivery
* Automated ServiceNow ticket creation for SSO Dashboard deployments
* Implement API Key Support on behalf of RPs
* [MOZILLIANS.ORG] Connect auditd to mozdef https://github.com/mozilla/parsys/issues/93
* Get SSO Dashboard launch-ready for community
   - Implement low- and med-risk alert UI https://github.com/mozilla-iam/sso-dashboard/issues/130
* Slack for Community access (next iteration of the current manual _NDA Pilot_)
* WorkDay/CIS integration
* [SSO Dashboard] (bugfix) Run access-proxy in front of SSO dashboard (https://github.com/mozilla-iam/sso-dashboard/issues/107)
* Move AirMo to 'hosted Auth0 Lock'
* SSO Dashboard improvements
   - UX: user favorites https://github.com/mozilla-iam/sso-dashboard/issues/92 / frequently used #98
   - Fix SSO with ParSys properties
* [AUTH0 LOCK] _Prototype new technology stack_
   - Sign Up UI design #67 
   - Decide on technology stack for Front-end and AWS Lambda (backend)
   - Create backlog items

## Logistics
* Out of Office
   * THU/FRI 13/14 JUL _CoSS Copenhagen Work Days_ (@a-kilroy @akatsoulas @comzeradd @m-branson @hmitsch)
   * TUE 18 JUL @LeoMcA on a flight to Malaga in the afternoon, and then working from the Costa del Sol
* CoSS share: **(?)** @comzeradd @m-branson focus mostly on CoSS, @akatsoulas supports

**Comment by gdestuynder** [comment]
@johngian @akatsoulas do you know when (and if you have an issue maybe) for mozillians.org to push the CIS profile (at least in dev)?
This is for:  "Merge demo functionality of CIS to mozillians.org"

**Comment by johngian** [comment]
We need to postpone this for next sprint. Is this something that's blocking CIS dev?

**Comment by gdestuynder** [comment]
its blocking releasing CIS in prod ("and dev")
